pkg/agent/endpoints/workload/handler.go

```diff
@@ -75,8 +77,20 @@ func (h *Handler) FetchJWTSVID(ctx context.Context, req *workload.JWTSVIDRequest
 			SpiffeId: spiffeID,
 			Svid:     svid,
 		})
+
+		spiffeIDsBuilder.WriteString(spiffeID)
+		if i < len(spiffeIDs)-1 {
+			spiffeIDsBuilder.WriteString(",")
+		}
 	}
 
+	metrics.IncrCounterWithLabels(
+		[]string{workloadApi, "fetch_jwt_svid"}, 1,
+		[]telemetry.Label{
+			{Name: "spiffe_id", Value: spiffeIDsBuilder.String()},
```

Comment:
`strings.Join(spiffeIDs, ",")` feels like a more natural choice and cuts down on the code quite a bit